### PR TITLE
kyma-cli 1.5.0 (new formula)

### PR DESCRIPTION
**Pull request by suleymanakbas91**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
👍

**Comment by chenrui333** [comment]
Thanks @suleymanakbas91 for your contribution to Homebrew! :tada: 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by chenrui333** [comment]
Bottle step has some issue, let's have another build.

**Comment by chenrui333** [comment]
@BrewTestBot test this please

File: Formula/kyma-cli.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/kyma-project/cli/"
+    bin_path.install Dir["*"]
+
+    cd bin_path do
+      system "dep", "ensure", "-vendor-only"
+      system "make", "build-darwin"
+      bin.install "bin/kyma-darwin"
```
Comment (SMillerDev) [inline Formula/kyma-cli.rb]: This can have a second parameter specifying the new name that would make the line below redundant.

**Review by SMillerDev — COMMENTED** [review]
One small tweak still

**Comment by chenrui333** [comment]
@suleymanakbas91 Any update on this PR?

File: Formula/kyma-cli.rb (hunk 1 above)
Comment (suleymanakbas91) [reply Formula/kyma-cli.rb]: Done. Thanks for the tip!

**Comment by suleymanakbas91** [comment]
@chenrui333 should be ready to go now :)

**Comment by chenrui333** [comment]
Thanks @suleymanakbas91 !!

**Pull request closed without merging**